templates/systemd/funkwhale-watch-imports.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Funkwhale Watch Imports Service ({{ funkwhale_watch_imports_identifier }})
+Requires={{ funkwhale_api_identifier }}.service
+After={{ funkwhale_api_identifier }}.service
+DefaultDependencies=no
+
+
+[Service]
+Type=simple
+ExecStart={{ devture_systemd_docker_base_host_command_docker }} exec {{ funkwhale_api_identifier }} \
+    funkwhale-manage import_files {{ funkwhale_watch_imports_library_id }} /srv/funkwhale/data/music --recursive --noinput --in-place --watch {{ funkwhale_watch_imports_process_extra_arguments | join(' ') }}
```

Comment:
I think this /srv file path is currently hardcoded in multiple places.

We should probably move it to a new dedicated variable (e.g. `funkwhale_data_music_path_in_container`) and all hardcoded references to be replaced with it.

Reply:
Done.